src/pull_automerge/core.clj

```diff
@@ -129,12 +138,17 @@
   (def org "ai-labs-team")
   (def repo "axiom-platform")
   (def label "Automerge")
+  (def priority "High Priority")
   (def options (generate-options token))
   (check-rate-limit options)
 
-  (def pulls (get-opened-labelled-issues org repo label options))
+  (def pulls (get-opened-labelled-issues org repo options label))
+
+  (def pull (or 
```

Comment:
I thought about putting this definition in the `get-oldest-issue-as-pull-request` function, but then I'd have to pass more variables into that since it didn't know what `priority` was or what the `filter-by-label` guy was